src/sorter.js

```diff
@@ -0,0 +1,32 @@
+class Sorter {
+  topFiveBars(results) {
+    const goodRating = this.#ratingFilter(results);
+    const orderedBars =  this.#orderByRating(this.#addPopulerRating(goodRating));
+
+    return orderedBars.slice(-5).reverse();
+  }
+
+  #ratingFilter(results) {
+    return results.filter( bar => bar.rating > 4 )
+  }
+
+  #addPopulerRating(results) {
```

Comment:
Can we just change the name of this method to be addPopularRating to be consistent with the others?

Reply:
Ah, its a typo, you mean?

Reply:
Ah right yes just that